assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -146,13 +147,20 @@ class SignupForm extends React.Component<Props, State> {
       userGuidelinesText
     } = this.props;
     const { privacyPolicyIsChecked, termsAndConditionsIsChecked, userGuidelinesIsChecked } = this.state;
-
     const legalContentsType = {
       termsAndConditions: 'termsAndConditions',
       privacyPolicy: 'privacyPolicy',
       userGuidelines: 'userGuidelines'
     };
 
+    const passwordRequirements = [
```

Comment:
So, THIS is configurable by a discussion preference. This should be fetched by the API that populates `debateData` in the store. And updated accordingly.

Reply:
Good first pass for default case, but can be overwritten, and then this would be wrong. The tricky part is that you need to expose this part of the API to the public. Meaning, the API will be called independent of logged in or not.

Reply:
We decided to merge this default configuration and add the "reste à faire" in a new ticket : https://bluenove.atlassian.net/browse/DEVAS-2130
I will talk with you to make a more precise description of this ticket